actor-testkit-typed/src/main/scala/org/apache/pekko/actor/testkit/typed/internal/ActorSystemStub.scala

```diff
@@ -137,7 +137,7 @@ import pekko.util.FutureConverters._
 
 @InternalApi private[pekko] object ActorSystemStub {
   object config {
-    // this is backward compatible with the old behaviour, hence it uses the loader used to load the test-kit
```

Comment:
Makes sense to be in line with the class names. Have we documented somewhere that we prefer American English over British English so we can move in that direction?